packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -173,21 +191,29 @@ export class Table extends Construct {
   private tablePartitionKey?: Attribute;
   private tableSortKey?: Attribute;
 
+  private readonly billingMode: BillingMode;
   private readonly tableScaling: ScalableAttributePair = {};
   private readonly indexScaling = new Map<string, ScalableAttributePair>();
   private readonly scalingRole: iam.IRole;
 
   constructor(parent: Construct, name: string, props: TableProps = {}) {
     super(parent, name);
 
+    this.billingMode = props.billingMode || BillingMode.Provisioned;
```

Comment:
This style is a bit sensitive. If `billingMode` had been a number, then `0` would evaluate to "false". Rather write: `props.billingMode !== undefined ? props.billingMode : BillingMode.Provisioned`